scale_build/packages/git.py

```diff
@@ -105,7 +105,7 @@ def checkout(self, branch_override=None, retries=3):
                 for cmd in map(lambda c: self.git_args + c, cmds):
                     cp = run(cmd, check=False)
                     if cp.returncode:
-                        failed = (f'{" ".join(cmd)}', f'{cp.stderr}', f'{cp.returncode}')
+                        failed = (f'{" ".join(cmd)}', f'{cp.stdout}', f'{cp.returncode}')
```

Comment:
Is this intentional ?

Reply:
Yes, commit message describes why. `run` implementation redirects `stderr` to `stdout` by default.